### PR TITLE
PA-1676 Update styling on parcel identification form.

### PR DESCRIPTION
**Pull request by devinleighsmith**
Update snapshots to prevent classnames from being unique every run.
Integrate new format for required fields from dev

### CODE REVIEW

**Comment by devinleighsmith** [comment]
@Fosol @goemen @jdtoombs @agalbrai I recommend reviewing only faa5b7e07e2328df0c94a47fc506cc02d5a424bb
because this PR includes some commits from dev.

How do you do this? Either choose the commit from the filter when viewing the files or go to this link:
https://github.com/bcgov/PIMS/pull/664/commits/faa5b7e07e2328df0c94a47fc506cc02d5a424bb

File: frontend/src/App.scss [hunk 1]
```diff
@@ -70,3 +70,7 @@ div.bootstrap-table {
 .form-check.required input {
   border: 1px solid $secondary-variant-color;
 }
+
+.form-group.required textarea {
```
Comment (devinleighsmith) [inline frontend/src/App.scss:74]: add support for required field formatting to text areas as well.

File: frontend/src/components/common/form/StepForm/context.tsx [hunk 2]
```diff
@@ -39,6 +39,7 @@ export const StepperFormProvider: React.FC<IStepperFormProviderProps> = ({
   };
 
   const gotoTab = (index: number) => {
+    debugger;
```
Comment (devinleighsmith) [inline frontend/src/components/common/form/StepForm/context.tsx:42]: this has been removed in the next commit, so safe to ignore here.

File: frontend/src/features/mapSideBar/SidebarContents/subforms/AssociatedLandReviewPage.tsx [hunk 3]
```diff
@@ -263,27 +264,35 @@ export const AssociatedLandReviewPage: React.FC<any> = (props: IReviewProps) =>
...
                             disabled={editInfo.valuation}
                           />
                         </Row>
                         <Row className="val-row">
-                          <Label>Appraised Value</Label>
-                          <FastCurrencyInput
-                            formikProps={formikProps}
-                            field={withNameSpace('financials.0.appraised.value', index)}
-                            disabled={editInfo.valuation}
+                          <Label>Total Assessed Value</Label>
```
Comment (devinleighsmith) [inline frontend/src/features/mapSideBar/SidebarContents/subforms/AssociatedLandReviewPage.tsx:283]: Fix some valuation formatting on the review page.

File: frontend/src/features/mapSideBar/SidebarContents/subforms/ParcelIdentificationForm.scss [hunk 4]
```diff
@@ -1,97 +1,65 @@
-.req {
-  color: red;
-  font-weight: bold;
-}
+@import '../../../../colors.scss';
 
-.postal {
+.parcel-identification {
+  max-width: 900px;
```
Comment (devinleighsmith) [inline frontend/src/features/mapSideBar/SidebarContents/subforms/ParcelIdentificationForm.scss:4]: This page was expanding the sidebar width before this.

File: frontend/src/features/mapSideBar/SidebarContents/subforms/ParcelIdentificationForm.scss [hunk 5]
```diff
@@ -1,97 +1,65 @@
...
-              font-size: smaller;
-            }
-            .tooltip-icon {
-              margin-top: 8px;
-              margin-left: 5px;
-            }
-          }
-        }
-      }
+    .section.disabled {
```
Comment (devinleighsmith) [inline frontend/src/features/mapSideBar/SidebarContents/subforms/ParcelIdentificationForm.scss:19]: don't allow any clicks on the disabled information section of the form.

File: frontend/src/features/mapSideBar/SidebarContents/subforms/ParcelIdentificationForm.scss [hunk 6]
```diff
@@ -1,97 +1,65 @@
...
-      display: inline-block;
-      width: 140px;
       text-align: right;
-      margin-right: 20px;
+      margin-right: 10px;
+      width: 125px;
     }
     .harmful {
-      background-color: #fae1e7;
+      margin: 0px auto;
```
Comment (devinleighsmith) [inline frontend/src/features/mapSideBar/SidebarContents/subforms/ParcelIdentificationForm.scss:29]: this provides horizontal centering.

File: frontend/src/features/mapSideBar/SidebarContents/subforms/ParcelIdentificationForm.tsx [hunk 7]
```diff
@@ -127,33 +145,36 @@ export const ParcelIdentificationForm: React.FC<IIdentificationProps> = ({
             nameSpace={withNameSpace('address')}
             hideStreetAddress
           />
-          <Row className="field-row">
+          <Form.Row>
```
Comment (devinleighsmith) [inline frontend/src/features/mapSideBar/SidebarContents/subforms/ParcelIdentificationForm.tsx:148]: I like using form.row to represent a group tags displayed on a single line. This actually provides very similar functionality to the previous commit, just with a className of form-row (but not needing to provide the classname each time).

File: frontend/src/features/mapSideBar/SidebarContents/subforms/ParcelIdentificationForm.tsx [hunk 8]
```diff
@@ -94,31 +99,44 @@ export const ParcelIdentificationForm: React.FC<IIdentificationProps> = ({
...
             </Col>
           </Row>
         </Col>
       </Row>
-      <Row noGutters>
+      <Row
+        noGutters
+        className={classNames(
+          'section',
+          getIn(formikProps.values, withNameSpace('address.line1')) === '' &&
```
Comment (devinleighsmith) [inline frontend/src/features/mapSideBar/SidebarContents/subforms/ParcelIdentificationForm.tsx:130]: disable the information form if none of the search fields have been filled in. I've replaced this check with a check for whether or not the hidden lat/lng fields or filled in my most recent code.

File: frontend/src/components/common/form/StepForm/SteppedForm.tsx [hunk 9]
```diff
@@ -170,11 +170,17 @@ export const SteppedForm = function<T extends object = {}>({
               title="Really Remove Associated Parcel?"
               message="Click OK to remove the association between this parcel and the current building."
               handleOk={() => {
-                if (onRemoveTab && tabToDeleteId !== undefined) {
+                if (values.tabs && onRemoveTab && tabToDeleteId !== undefined) {
```
Comment (devinleighsmith) [inline frontend/src/components/common/form/StepForm/SteppedForm.tsx:173]: added some comments describing what this logic is doing.

File: frontend/src/features/mapSideBar/SidebarContents/subforms/LandReviewPage.scss [hunk 10]
```diff
@@ -1,5 +1,6 @@
 .review-section {
   .row,
+  .form-row,
```
Comment (devinleighsmith) [inline frontend/src/features/mapSideBar/SidebarContents/subforms/LandReviewPage.scss:3]: using form-row again.

File: frontend/src/features/properties/components/GeocoderAutoComplete.tsx [hunk 11]
```diff
@@ -120,6 +119,7 @@ export const GeocoderAutoComplete: React.FC<IGeocoderAutoCompleteProps> = ({
               {...rest}
             />
           </TooltipWrapper>
+          {!!tooltip && <TooltipIcon toolTipId={`${field}-tooltip`} toolTip={tooltip} />}
```
Comment (devinleighsmith) [inline frontend/src/features/properties/components/GeocoderAutoComplete.tsx:122]: geocoder needs tooltip errors as well.

File: frontend/src/components/common/form/StepForm/context.tsx (hunk 2 above)
Comment (jdtoombs) [reply frontend/src/components/common/form/StepForm/context.tsx:42]: oops commented on other one lol got it

File: frontend/src/features/mapSideBar/SidebarContents/subforms/ParcelIdentificationForm.tsx (hunk 7 above)
Comment (jdtoombs) [reply frontend/src/features/mapSideBar/SidebarContents/subforms/ParcelIdentificationForm.tsx:148]: Ah gothcya, my bad. I think I was the one who swapped these out.

File: frontend/src/features/mapSideBar/SidebarContents/subforms/ParcelIdentificationForm.scss [hunk 12]
```diff
@@ -1,97 +1,65 @@
...
       width: 400px;
+      background-color: #fae1e7;
```
Comment (agalbrai) [inline frontend/src/features/mapSideBar/SidebarContents/subforms/ParcelIdentificationForm.scss:31]: do you want to maybe add this to colors.scss as "light-danger-color"?

**Review by jdtoombs — APPROVED** [review]

**Review by agalbrai — APPROVED** [review]

File: frontend/src/features/mapSideBar/SidebarContents/subforms/LandReviewPage.scss [hunk 13]
```diff
@@ -1,5 +1,6 @@
 .review-section {
```
Comment (jdtoombs) [inline frontend/src/features/mapSideBar/SidebarContents/subforms/LandReviewPage.scss:1]: I noticed the year values on the valuation section of buildings are a bit misaligned now. Do these changes cover that? If not all good I'll change it - just don't want to double up. 
image

Comment (devinleighsmith) [reply frontend/src/features/mapSideBar/SidebarContents/subforms/LandReviewPage.scss:1]: looks like it
image

**Pull request merged**